### DISCUSSION TITLE
openAIAuth _part_eight response 404

### DISCUSSION THREAD

**Discussion by FAIpang** [acheong08/ChatGPT#763] [General]
![8B3C9387-1285-4511-B696-931AE07C5BBB](https://user-images.githubusercontent.com/73394302/218990920-28da00aa-2db9-4a6e-81a3-9d486ed042a4.png)

I tried running the program, but the response returns 404 when this method is requested,

**Comment by acheong08** [comment]
Can't replicate

**Reply by FAIpang** [reply]
I don't know how to solve this problem, can you give me some advice，thank you。
